vm/src/vm.h

```diff
@@ -9,7 +9,7 @@
 /**
  * Executes the given Moshell bytecode.
  *
- * The bytecode will be immediately executed without any sanity checks.
+ * The bytecode will be immediately executed.
```

Comment:
```suggestion
 * The bytecode will be immediately executed without any sanity checks.
```
This is still true, if the generated bytecode is invalid, this is still undefined behavior.

Reply:
A sanity check is not a runtime check. The fact that a jump address is not verified, that a local size is not verified and a lot of things implies that there are no sanity checks.